### ISSUE TITLE
Need improvement in Datatables Server Side Processing

### ISSUE THREAD

**Issue by efriandika** [smnedelko/PixelAdmin#6]
Need improvement in datatables server side processing. Since, there is no style  for progress indicator. Currently, default progress bar indicator from datafables will break the table style.

**Comment by efriandika** [comment]
In order to solve this problem, I suggest you to add this script below into `js/src/extensions/datatables.js`

```
$.extend(true, $.fn.dataTable.ext.classes, {
     sProcessing: "dataTables_processing"
 });
```

**Comment by smnedelko** [comment]
Hi efriandika,

Thanks for the report. I will add this fix in the upcoming update.

**Comment by smnedelko** [comment]
Fixed in 2.3.0
